src/assets/style.css

```diff
@@ -428,8 +433,16 @@ input[type=radio]:checked + .checkbox {
   flex-shrink: 0;
 }
 
+.dashboard-options__button:focus {
+  outline: 0;
+}
+
 .dashboard-options__button-active {
-  background-color: #42e2f4;
+  filter: brightness(1.5);
+  background-color: rgba(120,120,120,0.8);
```

Comment:
Expected single space after "," in a single-line function    function-comma-space-after<br>Unexpected leading zero                                      number-leading-zero